dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.35/V2_35_11__Add_sequentialcounter_procedure.sql

```diff
@@ -0,0 +1,24 @@
+-- remove primary key on id
```

Comment:
@luciano-fiandesio / @stian-sandvold , please synchronize this Flyway migration version with the others PR's:

https://github.com/dhis2/dhis2-core/pull/5876/
https://github.com/dhis2/dhis2-core/pull/5885